<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>403</width>
    <height>709</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QStateMachine - example</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="7" column="0">
     <widget class="QLabel" name="lblUserInfo">
      <property name="text">
       <string>User info</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QGroupBox" name="gbMake">
      <property name="title">
       <string>Coffee maker</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QProgressBar" name="progressBar">
         <property name="value">
          <number>24</number>
         </property>
         <property name="textVisible">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pbCup">
         <property name="text">
          <string>Cup</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="11" column="0">
     <widget class="QPushButton" name="pbExit">
      <property name="text">
       <string>E&amp;xit</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QGroupBox" name="gbControlpanel">
      <property name="title">
       <string>Admin Menu</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QPushButton" name="pbRefill">
         <property name="text">
          <string>Refill</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="10" column="0">
     <widget class="QPlainTextEdit" name="stateLogger"/>
    </item>
    <item row="2" column="0">
     <widget class="QGroupBox" name="gbMoney">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>100</height>
       </size>
      </property>
      <property name="title">
       <string>Insert coin</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="2" column="1">
        <widget class="QLCDNumber" name="lcdNeeded">
         <property name="smallDecimalPoint">
          <bool>false</bool>
         </property>
         <property name="digitCount">
          <number>3</number>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="lblNeeded">
         <property name="text">
          <string>Money needed</string>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="QLCDNumber" name="lcdInserted"/>
       </item>
       <item row="2" column="2">
        <widget class="QLabel" name="lblInserted">
         <property name="font">
          <font>
           <pointsize>9</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Money insterted</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QPushButton" name="pb20c">
         <property name="text">
          <string>20C</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="pb50c">
         <property name="text">
          <string>50C</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2" colspan="2">
        <widget class="QPushButton" name="pbCancel">
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="9" column="0">
     <widget class="QLabel" name="lblStateLogger">
      <property name="text">
       <string>State logger</string>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QPlainTextEdit" name="userInfo"/>
    </item>
    <item row="0" column="0">
     <widget class="QGroupBox" name="gbType">
      <property name="title">
       <string>Select type of coffee</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="leftMargin">
        <number>8</number>
       </property>
       <property name="topMargin">
        <number>10</number>
       </property>
       <property name="rightMargin">
        <number>10</number>
       </property>
       <property name="bottomMargin">
        <number>10</number>
       </property>
       <item>
        <widget class="QPushButton" name="pbHotchocolate">
         <property name="text">
          <string>Hot Chocolate</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pbCappuccino">
         <property name="text">
          <string>Cappuccino</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pbEspresso">
         <property name="text">
          <string>Espresso</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="12" column="0">
     <widget class="QGroupBox" name="gbHidden">
      <property name="title">
       <string>Hidden</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="leftMargin">
        <number>10</number>
       </property>
       <property name="topMargin">
        <number>10</number>
       </property>
       <property name="rightMargin">
        <number>10</number>
       </property>
       <property name="bottomMargin">
        <number>10</number>
       </property>
       <item>
        <widget class="QPushButton" name="pbYes">
         <property name="text">
          <string>Yes</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pbNo">
         <property name="text">
          <string>No</string>
         </property>
         <property name="default">
          <bool>false</bool>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pbExit</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>172</x>
     <y>242</y>
    </hint>
    <hint type="destinationlabel">
     <x>172</x>
     <y>142</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
